Frontend/components/products/productsTable.js

```diff
@@ -39,7 +39,8 @@ const ProductsTable = () => {
                             <TableCell>ID</TableCell>
                             <TableCell>Product</TableCell>
                             <TableCell>Quantity</TableCell>
-                            <TableCell>Selling Price</TableCell>
+                            <TableCell align="right">Selling Price</TableCell>
+                            <TableCell></TableCell>
```

Comment:
Why do we need an empty cell? Has it something to do with formatting? :)

Reply:
I posted a screenshot as example above